### ISSUE TITLE
Functorch_dp_cifar10 Accuracy failed for key name bn1.bias.grad

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#502]
### 🐛 Describe the bug

torchbench_float16_training
xpu  train functorch_dp_cifar10               
E0626 12:46:28.252000 140124652623680 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.00107, (ref-fp64): 0.00007 and shape=torch.Size([64]). res.dtype: torch.float16, multiplier: 3.000000, tol: 0.001000
E0626 12:46:28.253000 140124652623680 torch/_dynamo/utils.py:1392] Accuracy failed for key name bn1.bias.grad
fail_accuracy

loading model: 0it [00:00, ?it/s]
loading model: 0it [00:01, ?it/s]

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by weishi-deng** [comment]
Similar to https://github.com/intel/torch-xpu-ops/issues/508

**Comment by chuanqi129** [comment]
close it due to duplicate
